android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIText.java

```diff
@@ -433,6 +436,18 @@ protected KrollDict getFocusEventObject(boolean hasFocus)
 	@Override
 	public boolean onEditorAction(TextView v, int actionId, KeyEvent keyEvent)
 	{
+		// TIMOB-23757: https://code.google.com/p/android/issues/detail?id=182191
+		if (Build.VERSION.SDK_INT < 24 && (getGravity() & Gravity.LEFT) != Gravity.LEFT) {
+			if (getNativeView() != null) {
+				ViewGroup view = (ViewGroup) getNativeView().getParent();
+				view.setFocusableInTouchMode(true);
+				view.requestFocus();
+			}
+			Context context = TiApplication.getInstance().getApplicationContext();
+			InputMethodManager inputManager = (InputMethodManager) context.getSystemService(Context.INPUT_METHOD_SERVICE);
+			inputManager.hideSoftInputFromWindow(tv.getWindowToken(), 0);
```

Comment:
[hideSoftInputFromWindow](https%3A%2F%2Fdeveloper.android.com%2Freference%2Fandroid%2Fview%2Finputmethod%2FInputMethodManager.html%23hideSoftInputFromWindow(android.os.IBinder%2C%20int)) `0` is used for the flags. I'm not using the `ResultReceiver`.